### PR TITLE
[VI] Translation of README.md, index.md, 01.md

### PR DESCRIPTION
**Pull request by JohnsonNguyen1820**

### CODE REVIEW

**Comment by skyil7** [comment]
Hello. @JohnsonNguyen1820 .
We have translation guidelines on our (wiki)[https://github.com/Atcold/pytorch-Deep-Learning/wiki/Translation-Guidelines]. Please check this out.

You should follow our PR naming convention.

Thank you for contribution.

File: docs/vi/index.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
+---
+Bố cục: Mặc định
+Tiêu đề: Học sâu
+Tác giả: Alfredo Canziani
+Tham khảo: Trang chủ
```
Comment (Atcold) [inline docs/vi/index.md:5]: You cannot translate the tags. These are read by my program. These `MD` files are the "source code" for my script, which converts them into `HTML` to be served as web pages.
Please, use 

```
---
layout: default
title: DEEP LEARNING
author: Alfredo Canziani
lang-ref: home
---
```

and replace "DEEP LEARNING" with Vietnamese. You need to add `lang: vi`, `translator:`, and `translation-date:` as well. Please, check again other languages, okay?
This is Korean, for example.

```
---
layout: default
title: 딥러닝
author: Alfredo Canziani
lang-ref: home
lang: ko
translation-date: 22 Mar 2020
translator: Gio
---
```

File: docs/vi/week01/01.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+Tham khảo: 01.md
+Tiêu đề: Tuần_01
```
Comment (Atcold) [inline docs/vi/week01/01.md:3]: Same as above, please, please, check other languages and how they've translated.
@xcastilla can you help?

**Review by Atcold — CHANGES_REQUESTED** [review]
Just a few changes.
Please, ask us to explain better if they don't make sense, okay?

File: docs/vi/week01/01.md (hunk 2 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/01.md:3]: Ok. I will correct it as soon as when I come back my office. I think I will translate all of them, so I was wrong when I translated this file.

File: docs/vi/index.md (hunk 1 above)
Comment (JohnsonNguyen1820) [reply docs/vi/index.md:5]: Ok. thank you for your annoucement so much! I will correct it again.

File: docs/vi/index.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
+---
+layout: defaut
+Title: Học sâu
```
Comment (Atcold) [inline docs/vi/index.md:3]: `title:` not `Title:`. My script won't understand capital letters. It's case sensitive.

File: docs/vi/week01/ch01.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+lang: vi
+lang-ref: ch01.md
+title: week01
```
Comment (Atcold) [inline docs/vi/week01/ch01.md:4]: Translate `week 01` to Vietnamese.

File: docs/vi/week01/ch01.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+translation-date: Sept 16, 2020
```
Comment (Atcold) [inline docs/vi/week01/ch01.md:5]: Use "16 Sep 2020" format.

File: docs/vi/README-VI.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+-->
+
+<!-- Vietnamese -->
+<!-- Các bạn có thể tìm thấy phiên bản Tiếng Việt của 'README.md' [Vi](https://github.com/JohnsonNguyen1820/pytorch-Deep-Learning/blob/patch-1/docs/vi/README-VI.md) tại đây!
+
+<!-- English - Mandarin - Korean - Spanish - Italian - Turkish - Japanese - Arabic - Vietnamse-->
+[🇬🇧](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/README.md) &nbsp; [🇨🇳](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/zh/README-ZH.md) &nbsp; [🇰🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ko/README-KO.md) &nbsp; [🇪🇸](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/es/README-ES.md) &nbsp; [🇮🇹](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/it/README-IT.md) &nbsp; [🇹🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/README-TR.md) &nbsp; [🇯🇵](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) &nbsp; [🇸🇦](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) &nbsp; [vi](https://github.com/JohnsonNguyen1820/pytorch-Deep-Learning/blob/patch-1/docs/vi/README-VI.md)
```
[the remark this answers is not in the document]
Comment (Atcold) [reply docs/vi/README-VI.md:40]: Still the same issue. French and Farsi are missing, there is no Vietnamese flag.

File: docs/vi/README-VI.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+🇯🇵 日本語版の `README.md` は  [ここ](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) にあります.
+-->
+
+<!-- Arabic -->
+<!--
+🇸🇦 النسخة العربية من ملف `README.md`  [ar](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) والموقع الالكتروني.
+-->
+
+<!-- Vietnamese -->
+<!-- Các bạn có thể tìm thấy phiên bản Tiếng Việt của 'README.md' [vi](https://github.com/JohnsonNguyen1820/pytorch-Deep-Learning/blob/patch-1/docs/vi/README-VI.md) tại đây!
```
Comment (Atcold) [inline docs/vi/README-VI.md:37]: Please, remove these lines.

**Review by Atcold — CHANGES_REQUESTED** [review]

**Comment by Atcold** [comment]
Can you please apply the suggestions I gave you?
This PR was opened 12 days ago.

**Comment by JohnsonNguyen1820** [comment]
Dear Atcold/pytorch-Deep-Learning,

I had corrected this file. Please check it to help me!

Thank you,
************
*Head of AI Department. NGUYEN TAN HUYNH*

*FPT TELECOM INTERNATIONAL .,LTD*
*VIETNAM ARTIFICIAL INTELLIGENCE COMMUNITY*
*Phone number:* 0903.894.995
*Email:* tanhuynhfu@gmail.com
            huynhnt183@gmail.com
            nguyentanhuynh.covid19@ehealth.gov.vn

File: docs/vi/README-VI.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (JohnsonNguyen1820) [reply docs/vi/README-VI.md]: I has changed "vi" to "vn".

File: docs/vi/index.md (hunk 3 above)
Comment (JohnsonNguyen1820) [reply docs/vi/index.md]: I corrected this error.

File: docs/vi/week01/ch01.md (hunk 5 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/ch01.md:5]: I got it

File: docs/vi/README-VI.md (hunk 7 above)
Comment (JohnsonNguyen1820) [reply docs/vi/README-VI.md]: I removed this lines

File: docs/vi/week01/ch01.md (hunk 4 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/ch01.md:4]: Can I use this format?

**Comment by JohnsonNguyen1820** [comment]
> Can you please apply the suggestions I gave you?
> This PR was opened 12 days ago.

File: docs/vi/README-VI.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+-->
+<!-- Japanese -->
+<!--
+🇯🇵 日本語版の `README.md` は  [ここ](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) にあります.
+-->
+
+<!-- Arabic -->
+<!--
+🇸🇦 النسخة العربية من ملف `README.md`  [ar](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) والموقع الالكتروني.
+-->
```
Comment (Atcold) [inline docs/vi/README-VI.md:34]: I asked you to remove these lines.

File: docs/vi/README-VI.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+<!-- English - Mandarin - Korean - Spanish - Italian - Turkish - Japanese - Arabic - Vietnamse-->
+[🇬🇧](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/README.md) &nbsp; [🇨🇳](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/zh/README-ZH.md) &nbsp; [🇰🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ko/README-KO.md) &nbsp; [🇪🇸](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/es/README-ES.md) &nbsp; [🇮🇹](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/it/README-IT.md) &nbsp; [🇹🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/README-TR.md) &nbsp; [🇯🇵](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) &nbsp; [🇸🇦](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) &nbsp; [vn](https://github.com/JohnsonNguyen1820/pytorch-Deep-Learning/blob/patch-1/docs/vi/README-VI.md)
```
Comment (Atcold) [inline docs/vi/README-VI.md:37]: Add the French 🇫🇷, Persian 🇮🇷, and Russian 🇷🇺 flags.
Add the Vietnamese 🇻🇳 flag.

File: docs/vi/index.md [hunk 10]
```diff
@@ -0,0 +1,303 @@
...
+    <tr>
+      <td rowspan="1">Thực hành</td>
+      <td><a href="en/week13/13-3">GCNs III</a></td>
+      <td>
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/slides/11%20-%20GCN.pdf">🖥️</a>
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/16-gated_GCN.ipynb">📓</a>
+        <a href="https://youtu.be/2aKXWqkbpWg">🎥</a>
+      </td>
+    </tr>
+  </tbody>
```
Comment (Atcold) [inline docs/vi/index.md:292]: Chapter 14 is missing.

File: docs/vi/index.md [hunk 11]
```diff
@@ -0,0 +1,303 @@
...
+        <a href="https://drive.google.com/open?id=1opT7lV0IRYJegtZjuHsKhlsM5L7GpGL1">🖥️</a>
+        <a href="https://drive.google.com/open?id=1sdeVBC3nuh5Zkm2sqzdScEicRvLc_v-F">🖥️</a>
+        <a href="https://youtu.be/ycbMGyCPzvE">🎥</a>
+      </td>
+    </tr>
+    <tr><td><a href="en/week06/06-2">RNNs and attention</a></td></tr>
+    <tr>
+      <td rowspan="1">Thực hành</td>
+      <td><a href="en/week06/06-3">Training RNNs</a></td>
+      <td>
```
Comment (Atcold) [inline docs/vi/index.md:144]: RNN slides are missing.

File: docs/vi/week01/ch01.md (hunk 5 above)
Comment (Atcold) [reply docs/vi/week01/ch01.md:5]: You didn't change it.

File: docs/vi/week01/ch01.md (hunk 4 above)
Comment (Atcold) [reply docs/vi/week01/ch01.md:4]: You're supposed to translate "Week 1" to Vietnamese.

**Review by Atcold — CHANGES_REQUESTED** [review]
Also, what is `ch01.md`? I think you want to rename that file to `01.md`. (Use `git mv` to rename a file.)
Please, check *all* comments. Go to GitHub, and read what I've written to you. If you keep ignoring my messages we'll never get to merge this PR.

File: docs/vi/week01/ch01.md (hunk 4 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/ch01.md]: > You're supposed to translate "Week 1" to Vietnamese.

I will use the same your format

File: docs/vi/week01/ch01.md (hunk 5 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/ch01.md]: > You didn't change it.

I corrected it

**Comment by JohnsonNguyen1820** [comment]
> Also, what is `ch01.md`? I think you want to rename that file to `01.md`. (Use `git mv` to rename a file.)
> Please, check _all_ comments. Go to GitHub, and read what I've written to you. If you keep ignoring my messages we'll never get to merge this PR.

Ok. I got it.

File: docs/vi/README-VI.md (hunk 8 above)
Comment (JohnsonNguyen1820) [reply docs/vi/README-VI.md:34]: I mean I must remove from +5 to 35, all right?

File: docs/vi/README-VI.md (hunk 9 above)
Comment (JohnsonNguyen1820) [reply docs/vi/README-VI.md]: I added the French, Persian and Russian
But How can I add the Vietnamese flag?

File: docs/vi/index.md (hunk 11 above)
Comment (JohnsonNguyen1820) [reply docs/vi/index.md:144]: I copy the 'en' file and translate it.

**Comment by JohnsonNguyen1820** [comment]
> > Also, what is `ch01.md`? I think you want to rename that file to `01.md`. (Use `git mv` to rename a file.)
> > Please, check _all_ comments. Go to GitHub, and read what I've written to you. If you keep ignoring my messages we'll never get to merge this PR.
> 
> Ok. I got it.

image

File: docs/vi/index.md [hunk 12]
```diff
@@ -0,0 +1,326 @@
+---
+layout: defaut
+title: Học sâu
```
Comment (Atcold) [inline docs/vi/index.md:3]: Can you use all-caps?

File: docs/vi/index.md [hunk 13]
```diff
@@ -0,0 +1,326 @@
...
+      <th>Tuần</th>
+      <th align="left">Định dạng</th>
+      <th align="left">Tiêu đề</th>
+      <th align="left">Tài liệu tham khảo</th>
+    </tr>
+  </thead>
+  <tbody>
+<!-- =============================== WEEK 1 ================================ -->
+    <tr>
+      <td rowspan="3" align="center"><a href="en/week01/01">①</a></td>
```
Comment (Atcold) [inline docs/vi/index.md:40]: This link is pointing to the the `EN` English version.
Please, check how other languages are pointing to their own pages.

File: docs/vi/index.md [hunk 14]
```diff
@@ -0,0 +1,326 @@
...
+      <td rowspan="2">Bài giảng</td>
+      <td><a href="week01/01-1">History and motivation</a></td>
```
Comment (Atcold) [inline docs/vi/index.md:42]: Same here and elsewhere.
Also, why is this written in English?

File: docs/vi/index.md [hunk 15]
```diff
@@ -0,0 +1,326 @@
...
+      <td rowspan="2">
+        <a href="https://drive.google.com/open?id=1Q7LtZyIS1f3TfeTGll3aDtWygh3GAfCb">🖥️</a>
+        <a href="https://www.youtube.com/watch?v=0bMe_vCZo30">🎥</a>
+      </td>
+    </tr>
+    <tr><td><a href="week01/01-2">Evolution and DL</a></td></tr>
```
Comment (Atcold) [inline docs/vi/index.md:48]: No translation?

File: docs/vi/week01/01-1.md [hunk 16]
```diff
@@ -0,0 +1,94 @@
+---
+lang-ref: ch.01-1
+lecturer: Yann LeCun
```
Comment (Atcold) [inline docs/vi/week01/01-1.md:3]: `lang: vi` is missing.

File: docs/vi/week01/01-1.md [hunk 17]
```diff
@@ -0,0 +1,94 @@
...
+title: Động lực của học sâu, lịch sử và cảm hứng của nó
+authors: Yunya Wang, SunJoo Park, Mark Estudillo, Justin Mae
+date: 27 Jan 2020
+translator: Huynh Nguyen
+translator date: 01 Oct 2020
```
Comment (Atcold) [inline docs/vi/week01/01-1.md:8]:
```suggestion
translator-date: 01 Oct 2020
```

File: docs/vi/week01/01-1.md [hunk 18]
```diff
@@ -0,0 +1,94 @@
...
+
+In 1947, Donald Hebb had the idea that neurons in the brain learn by modifying the strength of the connections between neurons. This is called hyper learning, where if two neurons are fired together, then the connection linked between them increases; if they don't fire together, then the connection decreases.
+
+Later in 1948, cybernetics were proposed by Norbert Wiener, which is the idea that by having systems with sensors and actuators, you have a feedback loop and a self-regulatory system. The rules of the feedback mechanism of a car all come from this work.
+
+In 1957, Frank Rosenblatt proposed the Perceptron, which is a learning algorithm that modifies the weights of very simple neural nets.
+
+Overall, this idea of trying to build intellectual machines by simulating lots of neurons was born in 1940s, took off in 1950s, and completely died in late 1960s. The main reasons for the field dying off in 1960 are:
```
Comment (Atcold) [inline docs/vi/week01/01-1.md:35]: Everything is in English here. What's going on?

File: docs/vi/week01/01.md [hunk 19]
```diff
@@ -0,0 +1,22 @@
+---
+lang: vi
+lang-ref: 01.md
+title: week01
```
Comment (Atcold) [inline docs/vi/week01/01.md:4]: Title needs translation.

File: docs/vi/week01/01.md [hunk 20]
```diff
@@ -0,0 +1,22 @@
+---
+lang: vi
+lang-ref: 01.md
```
Comment (Atcold) [inline docs/vi/week01/01.md:3]: What is this? Don't change its original value.

**Review by Atcold — CHANGES_REQUESTED** [review]
You need to update `docs/vi/_config.yml` as well.

**Comment by Atcold** [comment]
> > > Also, what is `ch01.md`? I think you want to rename that file to `01.md`. (Use `git mv` to rename a file.)
> > > Please, check _all_ comments. Go to GitHub, and read what I've written to you. If you keep ignoring my messages we'll never get to merge this PR.
> > 
> > 
> > Ok. I got it.
> 
> image

I didn't not ask to change that.
The file `01-1.md` has `lang-ref: ch.01-1`. Don't change these parameters, or you're going to break the website.

File: docs/vi/index.md (hunk 12 above)
Comment (JohnsonNguyen1820) [reply docs/vi/index.md:3]: All-caps mean?

File: docs/vi/index.md (hunk 13 above)
Comment (JohnsonNguyen1820) [reply docs/vi/index.md]: Ok Sir. I corrected it.

File: docs/vi/index.md (hunk 14 above)
Comment (JohnsonNguyen1820) [reply docs/vi/index.md:42]: I will translate the titles later.

File: docs/vi/week01/01-1.md (hunk 16 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/01-1.md:3]: I added 'lang: vi' to this content

File: docs/vi/index.md (hunk 15 above)
Comment (JohnsonNguyen1820) [reply docs/vi/index.md:48]: I will translate this title later.

File: docs/vi/week01/01-1.md (hunk 17 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/01-1.md]: I corrected it.

File: docs/vi/week01/01-1.md (hunk 18 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/01-1.md:36]: I only copied this contents and translating it.

File: docs/vi/week01/01.md (hunk 19 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/01.md:4]: I want to use this format (the same English version)

Comment (JohnsonNguyen1820) [reply docs/vi/week01/01.md:4]: If it needs, I will translate it. Please advise!

File: docs/vi/week01/01.md (hunk 20 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/01.md]: I corrected this missing.

**Comment by JohnsonNguyen1820** [comment]
> e parameters, or you're going to break the website.

Ok. I got it. I corrected it

**Comment by JohnsonNguyen1820** [comment]
> You need to update `docs/vi/_config.yml` as well.

I will update it on your file?

File: docs/vi/index.md (hunk 12 above)
Comment (Atcold) [reply docs/vi/index.md]: ALL-CAPS is the all-caps version of "all-caps".
The website title is "DEEP LEARNING" not "Deep learning".

File: docs/vi/index.md [hunk 21]
```diff
@@ -0,0 +1,326 @@
...
+      <th>Tuần</th>
+      <th align="left">Định dạng</th>
+      <th align="left">Tiêu đề</th>
+      <th align="left">Tài liệu tham khảo</th>
+    </tr>
+  </thead>
+  <tbody>
+<!-- =============================== WEEK 1 ================================ -->
+    <tr>
+      <td rowspan="3" align="center"><a href="week01/01">①</a></td>
```
Comment (Atcold) [inline docs/vi/index.md:40]: This link points to the English version.
Please, check how other languages have links pointing to their corresponding versions.

File: docs/vi/week01/01-1.md [hunk 22]
```diff
@@ -0,0 +1,94 @@
...
+
+The method of how to find it is computing gradient. For example, if we are lost in a smooth mountain at foggy night and want to go to the village in the valley. One way could be turning around and seeing which way the steepest way is to go down then take a small step down. The direction is (negative) gradient. With the assumption that the valley is convex, we could reach the valley.
+
+The more efficient way is called Stochastic Gradient Descent (SGD). Since we want to minimize average loss over the training set, we take one sample or small group of samples and calculate the error, then use gradient descent. Then, we take a new sample and get a new value for the error, then get the gradient which is a different direction normally. Two of the main reasons for using SGD are that it helps a model to converge fast empirically if the training set is very large and it enables better generalization, which means getting similar performance on various sets of data.
```
Comment (Atcold) [inline docs/vi/week01/01-1.md:62]: Missing translation.

File: docs/vi/week01/01-1.md [hunk 23]
```diff
@@ -0,0 +1,94 @@
...
+
+
+## Hierarchical representation of the Visual Cortex
+
+Experiments by Fukushima gave us an understanding of how our brain interprets the input to our eyes. In summary, it was discovered that neurons in front of our retina compress the input (known as contrast normalization) and the signal travels from our eyes to our brain. After this, the image gets processed in stages and certain neurons get activated for certain categories. Hence, the visual cortex does pattern recognition in a hierarchical manner.
+
+Experiments in which researchers poked electrodes in specific areas of the visual cortex, specifically the V1 area made researchers realize that certain neurons react to motifs that appear in a very small area in a visual field and similarly with neighbouring neurons and neighbouring areas in the visual field. Additionally, neurons that react to the same visual field, react to different types of edges in an organized manner (*e.g.* vertical or horizontal edges). It is also important to note that there's also the idea that the visual process is essentially a feed forward process. Hence, somehow fast recognition can be done without some recurrent connections.
```
Comment (Atcold) [inline docs/vi/week01/01-1.md:94]: Missing translation.

File: docs/vi/week01/01.md (hunk 19 above)
Comment (Atcold) [reply docs/vi/week01/01.md:4]: Use "Tuần 01" instead.

**Review by Atcold — CHANGES_REQUESTED** [review]
A few more edits. We're almost there.

**Comment by Atcold** [comment]
You need to update `docs/_config.yml` and add Vietnamese.

File: docs/vi/index.md (hunk 12 above)
Comment (JohnsonNguyen1820) [reply docs/vi/index.md]: Oh. I had changed this title.

File: docs/vi/week01/01.md (hunk 19 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/01.md]: Ok Sir. I corrected.

File: docs/vi/week01/01-1.md (hunk 22 above)
Comment (JohnsonNguyen1820) [reply docs/vi/week01/01-1.md]: I am still translating it.

File: docs/vi/index.md (hunk 21 above)
Comment (JohnsonNguyen1820) [reply docs/vi/index.md:40]: Ok Sir!

File: docs/vi/index.md [hunk 24]
```diff
@@ -0,0 +1,326 @@
...
+      <td>
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/slides/07%20-%20Regularisation.pdf">🖥️</a>
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/12-regularization.ipynb">📓</a>
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/slides/08%20-%20Bayesian%20NN.pdf">🖥️</a>
+        <a href="https://github.com/Atcold/pytorch-Deep-Learning/blob/master/13-bayesian_nn.ipynb">📓</a>
+        <a href="https://youtu.be/DL7iew823c0">🎥</a>
+      </td>
+    </tr>
+  </tbody>
+</table>
```
Comment (Atcold) [inline docs/vi/index.md:316]: There is week 15 now online.

**Review by Atcold — CHANGES_REQUESTED** [review]
`docs/vi/Week01/lecture01.sbv`, `docs/vi/Week01/practicum01.sbv` are in English. Please, translate it or remove it.
Please, update all other `README.md` to reflect the addition of the Vietnamese translation.

**Comment by JohnsonNguyen1820** [comment]
I am focusing and reparing for Deep Learning in Healtcare Image Analysis
Certification of NVIDIA, so I will update new translating contents later.

Vào 10:42, T.4, 25 Th11, 2020 Alfredo Canziani <notifications@github.com>
đã viết:

**Review by Atcold — APPROVED** [review]

**Comment by Atcold** [comment]
Congratulations, your translation is now online!
https://atcold.github.io/pytorch-Deep-Learning/vi/
Thanks! Now 14 weeks more, and we're done! 🥳

**Pull request merged**